### PR TITLE
Support loading swagger examples

### PR DESCRIPTION
**Pull request by PhoenixHe-msft**

### CODE REVIEW

File: lib/swagger/jsonLoader.ts [hunk 1]
```diff
@@ -106,11 +115,16 @@ export class JsonLoader implements Loader<Json> {
...
+    return this.mockNameMap[mockName];
+  }
+
   private async resolveRef(
     object: Json,
     pathArr: string[],
     rootObject: Json,
-    relativeFilePath: string
+    relativeFilePath: string,
+    skipResolveChildRef: boolean
```
Comment (raych1) [inline lib/swagger/jsonLoader.ts:127]: why don't make this as optional?

**Review by raych1 — COMMENTED** [review]
Please merge the change from develop.

File: lib/swagger/jsonLoader.ts (hunk 1 above)
Comment (PhoenixHe-msft) [reply lib/swagger/jsonLoader.ts:127]: It's a private function and I want to make it explicit to avoid bugs.

**Comment by PhoenixHe-msft** [comment]
@raych1 Can we merge this?

**Review by raych1 — APPROVED** [review]

**Pull request merged**